README.md

```diff
@@ -1,27 +1,44 @@
 This is a [Lightstep](https://lightstep.com) fork of https://github.com/GoogleCloudPlatform/microservices-demo
 
 # Hipster Shop: Cloud-Native Microservices Demo Application
+<p style="color:red">Can we change the name of the app and "rebrand" it to be about observability rather than microservices in the cloud?</p>
 
-This project contains a 10-tier microservices application. The application is a
-web-based e-commerce app called **“Hipster Shop”** where users can browse items,
-add them to the cart, and purchase them.
+- [Service Architecture](#service-architecture)
+- [Features](#features)
+- [Installation Options](#installation-options)
+- [Prerequisites](#prerequisites)
+  - [GKE Installation Only](#gke-installation-only)
+  - [Local Installation Only](#local-installation-only)
+  - [All Installations](#all-installations)
+- [GKE Installation](#gke-installation)
+- [Local Installation](#local-installation)
+- [(Optional) Deploying on a Istio-installed GKE cluster](#optional-deploying-on-a-istio-installed-gke-cluster)
+- [See Telemetry Data in LightStep](#see-telemetry-data-in-lightstep)
+- [Cleanup](#cleanup)
+- [Conferences featuring Hipster Shop](#conferences-featuring-hipster-shop)
 
-**Google uses this application to demonstrate use of technologies like
-Kubernetes/GKE, Istio, Stackdriver, gRPC and OpenCensus**. This application
-works on any Kubernetes cluster (such as a local one), as well as Google
+
+The application is a web-based e-commerce app called **Hipster Shop** where users can browse items,
+add them to the cart, and purchase them.  Google uses this application to demonstrate use of technologies like
+Kubernetes/GKE, Istio, Stackdriver, gRPC and OpenCensus.
+
+LightStep modified this application to demonstrate how to instrument for distributed tracing and monitoring using multiple tracing libraries, including OpenTelemetry, OpenTracing, and tracers from LightStep, Zipkin, and Jaeger.
+Use the app to learn and experiment with different tracing implementations and then run [LightStep](https://app.lightstep.com) to see your trace data in action.
+
+> 💡 Need a LightStep account? Sign up for our [**free trial**](https://go.lightstep.com/signup-dev.html)!
```

Comment:
Let's change "free trial" to "LightStep Developer" to avoid having even vauge trial / pricing terms on the README, where they might fall out of sync.